scripts/tests/udpsender_sdsm.py

```diff
@@ -0,0 +1,19 @@
+import socket
+import time
+import os
+
+# Currently set to oim-dev environment's ODE
+UDP_IP = os.getenv('DOCKER_HOST_IP')
+UDP_PORT = 44950
+MESSAGE = "0029250a010c0c0a301fa04edb8816396666f30c34000a00002c43e94bba4200020002ec00280002"
```

Comment:
suggestion (non-blocking): add an sdsm test to the generic udp sender script also

Reply:
Added!